omnibus/package-scripts/agent/postinst

```diff
@@ -91,23 +91,37 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
         fi
 
         # Setup SELinux policy and label if SELinux detected on the host
+        # FIXME: Refactor warning messages
         if command -v semodule >/dev/null 2>&1 && [ -f "$INSTALL_DIR/embedded/bin/system-probe" ]; then
-            echo "Loading SELinux policy module for datadog-agent."
+            echo "Loading SELinux policy module for system-probe."
             semodule -v -i $CONFIG_DIR/selinux/system_probe_policy.pp >/dev/null 2>&1
             if [ "$?" != "0" ]; then
-                echo "The SELinux policy module for datadog-agent is not compatible with this host."
-                echo "Please install or update the selinux-policy-default package and then reinstall the datadog-agent package to configure SELinux for the Datadog Agent."
-                echo "Skipping SELinux configuration."
+                echo "Couldn’t load system-probe policy."
+                echo "To be able to run system-probe on your host, please install or update the"
+                echo "selinux-policy-default and policycoreutils-python-utils packages."
```

Comment:
```suggestion
                echo "selinux-policy-default and policycoreutils-python (or policycoreutils-python-utils depending on your distribution) packages."
```
Shouldn't we mention both here as well?

Reply:
No need then. Sorry, it wasn't obvious to me that this script is only used on Debian.